Makefile

```diff
@@ -17,6 +17,7 @@ pre-commit: lint build test
 deploy: version
 	date
 	time gcloud app deploy app.yaml
+	gcloud app deploy cron.yaml
```

Comment:
not sure if I need the time here

Reply:
I haven't found the output from `time` useful, so I'm fine skipping that.